components/authz-service/server/v2/project_test.go

```diff
@@ -189,8 +189,8 @@ func TestGetProject(t *testing.T) {
 			grpctest.AssertCode(t, codes.InvalidArgument, err)
 			assert.Nil(t, resp)
 		}},
-		{"if the project id is invalid, returns 'invalid argument'", func(t *testing.T) {
-			resp, err := cl.GetProject(ctx, &api.GetProjectReq{Id: "no spaces allowed"})
```

Comment:
so these `no space allowed` would return not found right? Could we add tests that verify that behavior?

Reply:
since we're not validating at this level, an invalid name is the same as a not found. there are other tests for not found names, but they seemed functionally equivalent. What do think?